frontend/src/views/Main.vue

```diff
@@ -16,7 +16,10 @@
         cols="12"
         md="9"
       >
-        <city-map />
+        <city-map
```

Comment:
Da die Methoden zur Erstellung der geoJson-Informationen relativ umfangreich sind und diese die Logik zur Darstellung der Übersichtskarte beinhalten, wäre es besser wenn die Methoden, Variablen und die verwendete city-map in eine eigene Komponente für die Karte der Suchergebnisse ausgelagert wird. 

Diese neue Komponente wird dann in der Main-View verwendet.